src/core/server/plugins/types.ts

```diff
@@ -136,6 +136,15 @@ export interface PluginManifest {
    */
   readonly requiredPlugins: readonly PluginName[];
 
+  /**
+   * An optional list of the other plugins which are imported by the UI bundles
+   * of this plugin. If the plugins listed here are disabled their bundles will
+   * still be loaded into the browser. Required by the @kbn/optimizer to allow
+   * cross plugin imports. "core" and plugins listed in "requiredPlugins" do not
+   * need to be listed here.
+   */
```

Comment:
Is it optional? Suggested edit for clarity:

```suggestion
  /**
   * List of plugin ids that this plugin's UI code imports modules from that are 
   * not in `requiredPlugins`.
   *
   * @remarks
   * The plugins listed here will be loaded in the browser, even if the plugin is 
   * disabled. Required by `@kbn/optimizer` to support cross-plugin imports. 
   * "core" and plugins already listed in `requiredPlugins` do not need to be 
   * duplicated here.
   */
```

Reply:
It's optional if you don't import from other plugins, but I like what you said instead.